sound/soc/sof/sof-priv.h

```diff
@@ -395,6 +395,8 @@ struct snd_sof_dev {
 
 	/* power states related */
 	enum sof_d0_substate d0_substate;
+	/* flag to track if the intended power target of suspend is S0ix */
+	bool s0_suspend;
```

Comment:
commit message : "Add flag 's0_suspend' indicate if the system is entering S0ix"